Resources/Prototypes/status_effects.yml

```diff
@@ -43,3 +43,7 @@
 
 - type: statusEffect
   id: ForcedSleep #I.e., they will not wake on damage or similar
+
+- type: statusEffect
+  id: Courage
+  alert: Courage
```

Comment:
Does this do anything?

Reply:
Its the buff shown in the first picture, and yes it may need tweaking for balancing purposes